tianshou/data/buffer/her.py

```diff
@@ -15,13 +15,13 @@ class HERReplayBuffer(ReplayBuffer):
     observation is a dictionary with keys ``observation``, ``achieved_goal`` and
     ``desired_goal``. Currently support only HER's future strategy, online sampling.
 
-    :param int size: the size of the replay buffer.
+    :param size: the size of the replay buffer.
```

Comment:
If there is a reliable docstring converter, it could be done. But why? Is there a large benefit in it?

This would need to adjust the sphinx build, and maybe also things like doc8 config